v3/plugins/pizzafactory/xtend-lang/0.1.0/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: pizzafactory
+name: xtend-lang
+version: 0.1.0
+type: VS Code extension
+displayName: Xtend Language
+title: Language support for Xtend
+description: VS Code extension that provides Xtend Language Server
+icon: https://camino.pizzafactory.jp/images/logos/pizza-flat.svg
+firstPublicationDate: "2019-08-12"
+category: Language
+repository: https://github.com/PizzaFactory/xtend-ide-extensions/
+spec:
+  containers:
+    - image: "eclipse/che-remote-plugin-runner-java8:7.0.0-next"
```

Comment:
Consider using a more recent build of the plugin runner: `"docker.io/eclipse/che-remote-plugin-runner-java8:next"` is used in the redhat/java plugin.

Reply:
I see. I fixed them (here and others) and re-pushed to the branch.